uaclient/cli.py

```diff
@@ -1569,7 +1569,17 @@ def action_status(args, *, cfg):
     show_beta = args.all if args else False
     token = args.simulate_with_token if args else None
     active_value = ua_status.UserFacingConfigStatus.ACTIVE.value
-
+    if cfg.is_attached:
+        try:
+            if contract.is_contract_changed(cfg):
+                cfg.add_notice("", messages.NOTICE_REFRESH_CONTRACT_WARNING)
+        except exceptions.UrlError as e:
+            with util.disable_log_to_console():
+                logging.error(
+                    messages.UPDATE_CHECK_CONTRACT_FAILURE.format(
+                        reason=str(e)
+                    )
+                )
```

Comment:
@CalvoM @orndorffgrant  Should we have a warning message here on the `json` output ?

Reply:
Updated.

Reply:
@CalvoM I think this should only be a warning, right @lucasmoura?
```diff
                logging.warning(err_msg)
                event.warning(err_msg)
```